smac/epm/gaussian_process_mcmc.py

```diff
@@ -0,0 +1,275 @@
+
+import logging
+import george
+import emcee
+import numpy as np
+
+from copy import deepcopy
+
+from smac.epm.base_gp import BaseModel
+from smac.epm.gaussian_process import GaussianProcess
+from smac.utils import normalization
+
+logger = logging.getLogger(__name__)
+
+
+class GaussianProcessMCMC(BaseModel):
```

Comment:
If we use both GP classes, this one should inherit from the other or both should be merged.

Reply:
is there any code duplication if we don't merge both classes? it does not seem so.